mods/ra/rules/ai.yaml

```diff
@@ -82,7 +82,7 @@ Player:
 				CheckRadius: 10c0
 			Consideration@2:
 				Against: Enemy
-				Types: Vehicle, Tank, Infantry, Defense
+				Types: Vehicle, Tank, Infantry, Defense, Water
```

Comment:
These will only tell the AI to avoid boats, so I don't think it will fix #6366 in general?